Source: written by Pierre Lejeune <superheron@gmail.com>
Upstream: submitted, see issue 7
Reason: fix missing c standard library headers
--- beid-middleware-3.5.2-source-5800.orig/_src/eidmw/common/Config.cpp	2010-12-21 13:47:32.319494838 +0100
+++ beid-middleware-3.5.2-source-5800/_src/eidmw/common/Config.cpp	2010-12-21 13:56:41.046149505 +0100
@@ -20,6 +20,7 @@
 #include <iostream>
 #include <stdlib.h>
 #include <limits.h>
+#include <sys/stat.h>
 
 #include "Config.h"
 #include "Util.h"
--- eid-mw-3.6.0-scm/_src/eidmw/pkcs11/cal.cpp	2011-07-18 20:45:08.221041224 +0200
+++ eid-mw-3.6.0-scm/_src/eidmw/pkcs11/cal.cpp	2011-08-06 15:29:51.489415014 +0200
@@ -27,7 +27,7 @@
 #include "cal.h"
 #include "log.h"
 #include "cert.h"
-#include "tlvbuffer.h"
+#include "TLVBuffer.h"
 
 
 #ifndef WIN32
